apps/masterbots.ai/lib/hooks/use-mb-chat.ts

```
@@ -331,6 +335,36 @@ export function useMBChat(config?: MBChatHookConfig): MBChatHookCallback {
   }
 
+  const appendAsContinuousThread = async (userMessage: AiMessage | CreateMessage) => {
+    const optimisticUserMessage = { ...userMessage, id: randomThreadId.current }
+    const message = followingQuestionsPrompt(userMessage.content, messages.concat(allMessages))
+    userContentRef.current = userMessage.content
+
+    const createdThread = await createThread({
+      threadId: randomThreadId.current as string,
+      parentThreadId: activeThread?.threadId,
+      chatbotId: chatbot ? chatbot?.chatbotId : 0,
+      jwt: session!.user?.hasuraJwt,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)